var/spack/repos/builtin.mock/packages/cmake-client-dep/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+def check(condition, msg):
```

Comment:
I don't see how this is used/called. Packages which inherit `CMakePackage` can define a class method `check`. I don't think this is for that, and I don't see it being used.

Reply:
Cruft (I just copy and pasted the existing `cmake-client` package to start from), removed.